### PR TITLE
add GoogLeNet(Inception) in Example

### PR DESCRIPTION
**Pull request by burness**
add the googlenet according to the [caffe googlenet prototxt](https://github.com/BVLC/caffe/blob/master/models/bvlc_googlenet/train_val.prototxt). The only difference may be the padding of the each layer.

### CODE REVIEW

**Comment by aymericdamien** [comment]
Awesome! It was actually missing from tflearn repo, thanks for adding it!

**Comment by dlhseu** [comment]
Do you have auxiliary classifier softmax0 and softmax1 in this network as shown in CVPR paper Fig.3

**Comment by burness** [comment]
@dlhseu I'm sorry that i don't know how to implement the auxiliary softmax0 and softmax1. In tensorflow, it is easy to do with `sess.run([softmax3, softmax0, softmax2])` after build the network. However, in tflearn  i don't know how to do with it 😆 . And I think that it make no difference.
